File: apis/backend/src/main/java/com/tms/api/objects/viewmodels/organization/OrganizationViewModel.java
package com.tms.api.objects.viewmodels.organization;

import com.tms.api.objects.models.organization.OrganizationDetailModel;
import com.tms.api.objects.models.organization.OrganizationModel;

public record OrganizationViewModel(
    OrganizationModel organization,
    OrganizationDetailModel organizationDetail
) {}

File: ticket-managment-system-web/src/app/core/view-models/organization/organization.view-model.ts
import { OrganizationDetailModel, OrganizationModel } from "../../models";

export class OrganizationViewModel {
    public organization!: OrganizationModel;
    public organizationDetail!: OrganizationDetailModel;

    constructor(organization: Partial<OrganizationModel> = {}) {
        Object.assign(this, organization);
    }
}